sonic-xcvrd/scripts/xcvrd

```diff
@@ -472,12 +482,14 @@ def main():
     # Start main loop to listen to the SFP change event.
     # The state migrating sequence:
     # 1. When the system starts, it is in "INIT" state, calling get_transceiver_change_event
-    #    with RETRY_PERIOD_FOR_SYSTEM_READY_MSECS as timeout for as many as RETRY_TIMES_FOR_SYSTEM_READY
-    #    times
+    #    with RETRY_PERIOD_FOR_SYSTEM_READY_MSECS as timeout for before reach RETRY_TIMES_FOR_SYSTEM_READY
+    #    times, otherwise it will transmit to "EXIT" state
```

Comment:
I believe the word you're looking for is "transition", not "transmit". Can you please change this throughout the file?

Reply:
thanks for the suggestion, fixed.